scripts/simulation_parameters/microbe_stage/bio_processes.json

```diff
@@ -160,7 +160,7 @@
         "inputs": {
             "carbondioxide": 0.09,
             "oxygen": 0.21,
-            "iron": 0.12
+            "iron": 0.024
```

Comment:
I don't think iron needs reducing, it already generated plenty of energy and iron is quite common.

Reply:
You're right, It probably shouldnt be reduced nearly as much as the others, I will test that feature more specifically and see if it should be reduced as much, if at all.